cmd/kite/oncall/cmd.go

```diff
@@ -101,7 +101,10 @@ func oncallHandler(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	utils.InfoLogger.Print("Initializing on-call view")
-	initOncallUI(&tui, onCallUsers)
+	for idx, x := range onCallLayers {
```

Comment:
Same here as well, what's `idx`?

Reply:
Idx refers to the index of the current OnCallLayer item. It is used to assign pagenumbers such that each oncall layer page is unique